include/numerics/type_vector.h

```diff
@@ -1053,6 +1054,8 @@ T solid_angle(const TypeVector<T> & v01,
               const TypeVector<T> & v02,
               const TypeVector<T> & v03)
 {
+  using std::atan;
+
```

Comment:
So what do the `using` statements do exactly? Just ensure that the `std::` version of the function takes part in the template argument deduction process?

Reply:
Some of the `<cmath>` functions like [atan](https://en.cppreference.com/w/cpp/numeric/math/atan.html) are templated, so I think you should use exactly that language :rofl: